src/app/register/components/form-personal/form-personal.component.html

```diff
@@ -0,0 +1,160 @@
+      <mat-label i18n="@@register.firstName">First name</mat-label>
+      <input formControlName="givenNames" matInput />
+      <mat-error
+        *ngIf="form.hasError('required', 'givenNames')"
+        i18n="@@register.firstNamePassword"
+      >
+        Please enter your first/given name
+      </mat-error>
+      <mat-error
+        *ngIf="form.hasError('notPattern', 'givenNames')"
+        i18n="@@register.invalidName"
+      >
+        Invalid name characters or format
+      </mat-error>
+      <mat-error *ngIf="form.hasError('backendError', 'givenNames')">
+        <div *ngFor="let error of form.getError('backendError', 'givenNames')">
+          <app-backend-error [errorCode]="error"></app-backend-error>
+        </div>
+      </mat-error>
+    </mat-form-field>
+
+    <button
+      mat-icon-button
+      aria-label="info about names"
+      type="button"
+      [mdePopoverTriggerFor]="namesPopover"
+      mdePopoverTriggerOn="click"
+      #namesPopoverTrigger="mdePopoverTrigger"
+    >
+      <mat-icon>info</mat-icon>
+    </button>
+  </div>
+  <mat-form-field appearance="outline">
+    <mat-label
+      ><ng-container i18n="@@register.lastName">Last name</ng-container>
+      <ng-container i18n="@@register.optional"
+        >(Optional)</ng-container
+      ></mat-label
+    >
+    <input formControlName="familyNames" matInput />
+    <mat-error
+      *ngIf="form.hasError('notPattern', 'familyNames')"
+      i18n="@@register.invalidName"
+    >
+      Invalid name characters or format
+    </mat-error>
+    <mat-error *ngIf="form.hasError('backendError', 'familyNames')">
+      <div *ngFor="let error of form.getError('backendError', 'familyNames')">
+        <app-backend-error [errorCode]="error"></app-backend-error>
+      </div>
+    </mat-error>
+  </mat-form-field>
+
+  <ng-container formGroupName="emails">
+    <mat-form-field appearance="outline">
+      <mat-label i18n="@@register.primaryEmail">Primary email</mat-label>
+      <input formControlName="email" matInput />
+      <mat-error
+        *ngIf="form.hasError('required', 'emails.email')"
+        i18n="@@register.primaryEmailRequired"
+      >
+        An email is required
+      </mat-error>
+      <mat-error
+        *ngIf="
+          this.emails.hasError('email', 'email') ||
+          this.emails.hasError('pattern', 'email')
+        "
+        i18n="@@register.invalidEmail"
+      >
+        Invalid email format
+      </mat-error>
+
+      <mat-error *ngIf="this.emails.hasError('backendError', 'email')">
+        <div
+          *ngFor="let error of this.emails.getError('backendError', 'email')"
+        >
+          <app-backend-error [errorCode]="error"></app-backend-error>
+        </div>
+      </mat-error>
+    </mat-form-field>
+    <mat-form-field appearance="outline">
+      <mat-label i18n="@@register.confirmEmail"
+        >Confirm primary email</mat-label
+      >
+      <input formControlName="confirmEmail" matInput (paste)="(false)" />
```

Comment:
@leomendoza123 I will remove the 'paste' attribute since we already have the validation if they are not the same email's and some users have very long email's

Reply:
This is a feature required by Paula, she doesn't want to allow users to paster the email on the confirmation field, this is to make sure the user writes the email there.